src/utils/src/lib.rs

```diff
@@ -1,8 +1,16 @@
 // Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: Apache-2.0 OR BSD-3-Clause
+#[macro_use]
+extern crate enum_display_derive;
```

Comment:
We are trying to keep the external dependencies to a minimum, can we remove this one?